tests_v2/test_tmux_compat_geometry.py

```
@@ -0,0 +1,247 @@
+import os
+import subprocess
+import sys
+import time
+from pathlib import Path
+from typing import List
+
+sys.path.insert(0, str(Path(__file__).parent))
+from cmux import cmux, cmuxError
+
+
+SOCKET_PATH = os.environ.get("CMUX_SOCKET", "/tmp/cmux-debug.sock")
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-debug.sock"

(S108)